server/tinylicious/src/app.ts

```diff
@@ -89,10 +76,12 @@ export function create(
 			);
 		}),
 	);
-<<<<<<< HEAD
->>>>>>> main
-=======
->>>>>>> main
+	app.use(
+		Router().post("/task-list-hook", (req, res) => {
```

Comment:
Probably use a path that's not specific to task-list -- maybe this should be `"/broadcast-signal"` or similar?